### DISCUSSION TITLE
Reading UEFI http boot uri in ipxe shell.

### DISCUSSION THREAD

**Discussion by durgeshm87** [ipxe/ipxe#1075] [General]
I am loading the ipxe.efi using UEFI http boot option in bios and would like to read the URI in ipxe shall to connect back to the http server to chainload next set of scripts. Please let me know if a option exists to read the UEFI http boot URI

**Comment by epacua** [comment]
If you enabled and know the code, you can inspect its values via shell. On my example below, I enabled some ipxe-encap-opts (175) and I can inspect its values along with standard dhcp opts like code 3 (router).
![ipxe_000](https://github.com/ipxe/ipxe/assets/15176373/7fc8e7bd-e2f5-4618-a3ac-4c2b49fc15c8)
